Plots/Satellites/NCL_sat_1.py

```diff
@@ -71,7 +97,15 @@
 # countour label to find coordinate (which can be found in bottom left of figure window)
 
 # low pressure contour levels- these will be plotted as a subscript to an 'L' symbol
-lowClevels = [(51.54, 169.59), (74.78, 4.54), (60.12, -57.0)]
+
+#lowClevels = [(51.54, 169.59), (74.78, 4.54), (60.12, -57.0)]
+
+coord956 = findLowPressureCoords(956)
```

Comment:
By specifying the low pressure value as an argument, it assumes that the user knows where the low pressure areas are. Try reworking this function to find low pressure areas for the user instead.